man/systemd.special.xml

```diff
@@ -357,6 +358,14 @@
             more discussion.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><filename>initrd-cryptsetup.target</filename></term>
+          <listitem>
+            <para>A target that pulls in setup services for all encrypted block devices.
+              See <filename>cryptsetup.target</filename> and <filename>remote-cryptsetup.target</filename>
+              for the real root equivalents.</para>
```

Comment:
"real root"? maybe say for "for the equivalents for regular (i.e. non-initrd) and remote encrypted volumes" instead

Reply:
I can do this if you'd prefer, though I couldn't find uses of either "regular" or "non-initrd" for this. I did find multiple references to "real root" though.